README.md

```diff
@@ -4,73 +4,70 @@
-| [Reactive Cache with Redis](./r2dbc/boot-r2dbc-reactive-cache)                                                          | Demonstrates caching reactive database operations in Redis to boost performance.                                                                                                                                                                                                                                                  | Completed |
-| [Reactive Application](./r2dbc/boot-r2dbc-sample)                                                                       | Illustrates an end-to-end reactive CRUD workflow in Spring Boot using R2DBC.                                                                                                                                                                                                                                                      | Completed |
-| [BackgroundJobs and Scheduling using Jobrunr](./scheduler/boot-scheduler-jobrunr)                                       | Sets up background job scheduling with [Jobrunr](https://www.jobrunr.io/en/) for asynchronous task execution.                                                                                                                                                                                                                     | Completed |
-| [Scheduling using Quartz](./scheduler/boot-scheduler-quartz)                                                            | Showcases the [Quartz Scheduler](https://www.quartz-scheduler.org/), providing a Thymeleaf UI to pause, resume, create, and delete scheduled tasks.                                                                                                                                                                               | Completed |
-| [Scheduling using Database Distributed Locks with ShedLock](./scheduler/boot-scheduler-shedlock)                        | Demonstrates scheduling while ensuring only one active job running at a time via ShedLock.                                                                                                                                                                                                                                        | Completed |
-
-
-For More info about this repository, Please visit [here](https://rajadilipkolli.github.io/my-spring-boot-experiments/)
+| Name                                                                                                                    | Description 		                                                                                              |
+|-------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
```

Comment:
@coderabbitai, can you please help me with updating this READMe.md file with information of all modules and submodules present in this folder in the existing tabular format, please add if any module info is missing